src/types/Roi.ts

```diff
@@ -57,11 +57,13 @@ export interface Roi<TData = unknown> extends Box {
   /**
    * Style of the ROI box when not being selected or edited.
    */
-  style: CSSProperties;
+  style?: CSSProperties;
+  className?: string;
   /**
    * Style applied to the ROI box when being selected.
    */
-  selectedStyle: CSSProperties;
+  selectedStyle?: CSSProperties;
+  selectedClassname?: string;
```

Comment:
Could you please add a brief tsdoc to the new props? It's worth mentioning that this className corresponds to the ROI className